content/post/ds_advice.md

```diff
@@ -0,0 +1,59 @@
+
+To the point above about learning the overlap - understand that linear regression can be formulated in multiple ways, each with its own nuance. For example, formulating LR as a statistical problem involves thinking probabilistically, talking about residuals and making assumptions around independence, homoscedasticity and Normality. It also involves minimizing a log likelihood function and dovetails nicely into conversations around [Bayesian inference](https://en.wikipedia.org/wiki/Bayesian_linear_regression), correlations, hypothesis testing, etc... There is a lot that falls out of thinking of LR from a statistics perspective.
+
+Formulating LR as a computer scientist would doesn't involve any of the above! It simply involves solving for a set of weights that minimizes the squared error between your estimate and the observation. No assumptions on Normality or independence or constant variance required. You end up with the same coefficients in both approaches, but now since you aren't making any assumptions on your data, you lose the ability to make inferences on the coefficients. This may or may not be useful to you, but, per the idea of learning the overlap, it's important to know that differences in these approaches exist.
+
+A similar example exists when learning logistic regression. The statisticians have their approach to formulating this problem which involves assumptions on residuals and objects called logit and GLMs. Computer scientists use words like cross-entropy loss and sigmoid. Both approaches get you to the same set of weights which minimize the same error function, but one approach allows you to calculate odds-ratios and confidence intervals around your coefficients and the other is a basic building block of learning about neural networks. Learn about how other fields formulate the same problem, in part to know how it works, and in part to be able to read the literature as you learn more. Data Science is built upon various fields each with their own problem formulation and terminology. Don't get confused.
+
+This stuff all builds on itself - don’t take short cuts. Understand what you’re doing and why. Learn the basics, but learn them well.
+
+<h3>Build some business intuition</h3>
+
+Learn basic analytics. This includes cohort modeling, attribution and funnel analysis and how an A/B testing works. Know what a marketing function does and why they (should) work closely with the Analytics team. Why should Product managers care about sample sizes? It's not rocket science so know this stuff because you'll be expected to. I won't harp on this too much because most of it will come on the job. Don't rush it, but keep it in mind when you're building a dashboard or querying a database at the office. It's harder to learn this stuff on your own compared to Python basics or how Decision Trees work.
```

Comment:
You could throw in some advice to learn some basic business fundamentals like how to read an income statement. I know that very few people do that, but for me personally that's been really helpful in linking my own work to how the business actually makes money.

Reply:
Good point. We're having a lot of conversations here in trying to understand our business levers and their financial impact.